integration-test/src/test/java/org/jboss/pnc/integration/endpoints/ProductMilestoneEndpointTest.java

```diff
@@ -87,6 +103,7 @@ public class ProductMilestoneEndpointTest {
     private static ProductMilestone milestone;
     private static String milestoneId;
     private static ProductMilestone milestone2;
+    private static BPMWireMock bpm;
```

Comment:
TODO: Discussion with @janinko about using BPMWireMock together with https://github.com/before & @after, rather than @BeforeClass & @afterclass

Reply:
I think we've already talked about this?

Reply:
yessir, you're right